ijkmedia/ijkplayer/ff_ffplay_def.h

```diff
@@ -799,6 +787,10 @@ inline static void ffp_notify_msg3(FFPlayer *ffp, int what, int arg1, int arg2)
     msg_queue_put_simple3(&ffp->msg_queue, what, arg1, arg2);
 }
 
+inline static void ffp_notify_msg4(FFPlayer *ffp, int what, int arg1, int arg2, void *obj) {
```

Comment:
the other 3 notify_msg implementations cannot send objects to java, so I add a new notify_msg to do this. this function is called in ff_ffplay.c to send subtitle text to java.